### ISSUE TITLE
Integrating Multi-Display Functionality into Appium Driver

### ISSUE THREAD

**Issue by ashirapara** [appium/appium-uiautomator2-driver#750]
The recent update to test-uiautomator (version 2.3.0) brings support for interacting with objects across multiple displays in Android applications, along with enhanced methods for managing secondary displays ([source](https://developer.android.com/jetpack/androidx/releases/test-uiautomator#2.3.0)).

In line with this progress, both appium-uiautomator2-server (version 7.0.1) and appium-uiautomator2-driver (version 3.0.1) have been updated to incorporate these changes ([source 1](https://github.com/appium/appium-uiautomator2-server/compare/v7.0.0...v7.0.1#diff-51a0b488f963eb0be6c6599bf5df497313877cf5bdff3950807373912ac1cdc9R92), [source 2](https://github.com/appium/appium-uiautomator2-driver/compare/v3.0.0...v3.0.1#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R62)).

In light of these enhancements, it would be beneficial to extend support for multi-display interactions through the Appium driver. By providing intuitive APIs for managing multiple displays, users can seamlessly navigate and interact with applications across different screens.

Therefore, I propose incorporating multi-display support into the Appium driver, empowering users to efficiently handle multiple devices with ease.

**Comment by KazuCocoa** [comment]
Yes, it could be. I don't recall what PR/issue, but we discussed such as topic before. We're always welcome to create PRs for it if you already have ideas/implementation about that, btw.
e.g. https://github.com/appium/appium-uiautomator2-driver?tab=readme-ov-file#mobile-screenshots

**Comment by cosminstirbu** [comment]
@KazuCocoa we are also interested in this functionality.

Has there been any progress in this area? Any POCs or branches that we can test?

Is this being discussed other issues?
